src/main/java/org/jabref/gui/search/GlobalSearchBar.java

```diff
@@ -325,7 +322,7 @@ private void updateResults(int matched, TextFlow description, boolean grammarBas
     }
 
     private void setHintTooltip(TextFlow description) {
-        if (Globals.prefs.getBoolean(JabRefPreferences.SHOW_ADVANCED_HINTS)) {
+        if (preferencesService.getGeneralPreferences().shouldShowAdvancedHints()) {
```

Comment:
I think it would make sense to add it to the search preferences as well

Reply:
Originally the idea about the advanced hints options was to make it reusable for other hints as well. But in fact, it has only been used also for another (somewhat silly) hint in the network tab in the preferences dialog. So im not sure about this... Maybe in the future, we could evolve that feature to some annoying tip of the day feature? :wink:

Reply:
I have no strong opinion on this. So then just leave it as is